src/renderer.ts

```diff
@@ -153,6 +156,430 @@ export class DefaultCellRenderer implements ICellRendererFactory {
   }
 }
 
+
+class HeatmapCellRenderer implements ICellRendererFactory {
+
+  createSVG(col: Column, context: IDOMRenderContext): ISVGCellRenderer {
+    const multiValueColumn: MultiValueColumn = (<MultiValueColumn>col);
```

Comment:
just request the target type already as argument type `col: MultiValueColumn` will be done by other renderers, too